### PR TITLE
draft first webid-provider tester

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
It's running, although it's failing against node-solid-server (the server returns a 404 on `/.well-known/openid-configuration`, but I think that's a configuration issue on the server side, not a problem with the test.

**Comment by Potherca** [comment]
Regarding `/.well-known/openid-configuration`, I haven't run node-solid-server, but looking at the code I am inclined to agree.

The [.well-known folder](https://github.com/solid/node-solid-server/tree/HEAD/default-templates/server/.well-known) only has an ACL entry and the only mention I could find in the code is [routing that has been commented out](https://github.com/solid/node-solid-server/blob/HEAD/lib/api/authn/webid-oidc.js#L113).

File: servers/node-solid-server/.db/oidc/op/provider.json [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    "client_credentials"
+  ],
+  "subject_types_supported": [
+    "public"
+  ],
+  "id_token_signing_alg_values_supported": [
+    "RS256",
+    "RS384",
+    "RS512",
+    "none"
```
Comment (poef) [inline servers/node-solid-server/.db/oidc/op/provider.json:34]: I haven't seen the code, but the existence of the "none" algorithm in JWT is generally seen as a security blunder. So wouldn't it be better to explicitly not support it in the test suite?

Comment (michielbdejong) [reply servers/node-solid-server/.db/oidc/op/provider.json:34]: Good point! This is in the default config of node-solid-server.
I'm not sure yet which algorithms a server should support as a minimum, but the test suite should test that "none" is not supported (the DPop spec mentions that).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+The goal of these test suites is to test which open source
+Personal Data Store implementations comply with which protocol specs.
+
+## Test Suite Summary in Docker
+
+To summarize the test results, we have opted to build them in Docker
+(this part was copied from [Solid's test-suite](https://github.com/solid/test-suite)),
+and to run them within that. This documents how to run it:
+
+Prerequisites: [Docker](https://docs.docker.com/install/)o
```
[the remark this answers is not in the document]
Comment (michielbdejong) [reply README.md:14]: :+1:

**Pull request merged**